pom.xml

```diff
@@ -37,6 +37,12 @@
             <version>${webdrivermanager.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
This can be removed since you removed the scope from above dependency.

Reply:
done